qds_sdk/cloud/azure_cloud.py

```diff
@@ -0,0 +1,206 @@
+        compute_creds = compute_config.add_mutually_exclusive_group()
+        compute_creds.add_argument("--enable_account_compute_creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_true",
+                                   default=None,
+                                   help="to use account compute credentials")
+        compute_creds.add_argument("--disable_account_compute_creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_false",
+                                   default=None,
+                                   help="to disable account compute credentials")
+        compute_config.add_argument("--compute-client-id",
+                                    dest="compute_client_id",
+                                    default=None,
+                                    help="client id for azure cluster")
+        compute_config.add_argument("--compute-client-secret",
+                                    dest="compute_client_secret",
+                                    default=None,
+                                    help="client secret key for azure cluster")
+        compute_config.add_argument("--compute-tenant-id",
+                                    dest="compute_tenant_id",
+                                    default=None,
+                                    help="tenant id for azure cluster")
+        compute_config.add_argument("--compute-subscription-id",
+                                    dest="compute_subscription_id",
+                                    default=None,
+                                    help="Subscription id for azure cluster")
+
+
+
+        # location settings parser
+        location_group = argparser.add_argument_group("location config settings")
+        location_group.add_argument("--location",
+                                    dest="location",
+                                    default=None,
+                                    help="location for azure cluster")
+
+        # network settings parser
+        network_config_group = argparser.add_argument_group("network config settings")
+        network_config_group.add_argument("--vnet-name",
+                                          dest="vnet_name",
+                                          help="vnet name for azure", )
+        network_config_group.add_argument("--subnet-name",
+                                          dest="subnet_name",
+                                          help="subnet name for azure")
+        network_config_group.add_argument("--vnet-resource-group-name",
+                                          dest="vnet_resource_group_name",
+                                          help="vnet resource group name for azure")
+        network_config_group.add_argument("--bastion-node-public-dns",
```

Comment:
arguments like bastion-node-public-dns, persistent-security-groups, master-elastic-ip are specific to aws I guess. Please verify and remove them from here. Also check arguments passed in oracle cloud.